app/models/publication.rb

```diff
@@ -438,6 +451,7 @@ def doi_url_path
       field(:created_at) { read_only true }
       field(:updated_at) { read_only true }
       field(:updated_by_user_at) { read_only true }
+      field(:preferred_file_version_none_email_sent)
```

Comment:
It might make sense now to group this and "preferred_version" into a group called "Preferred Version".  It would probably also be useful to add this to the RailsAdmin "list", so admins can filter on it.